packages/react/src/patterns/sub-patterns/ContentGroup/__stories__/ContentArrayGroup.stories.js

```diff
@@ -19,5 +19,9 @@ storiesOf('Patterns (Sub-Patterns)|ContentGroup', module)
       'Lorem ipsum dolor sit amet.'
     );
 
-    return <ContentGroup heading={copy} />;
+    return (
+      <div className="content-group-story">
+        <ContentGroup heading={copy} />
+      </div>
```

Comment:
Could we leverage the grid here instead? Similar to how we're doing in `FeatureCard`?

```html
<div class="bx--grid ">
  <div class="bx--row">
    <div class="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
      ...
    </div>
  </div>
</div>
```